v2/bundle/x509bundle/bundle_test.go

```diff
@@ -0,0 +1,216 @@
+package x509bundle
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_New(t *testing.T) {
```

Comment:
Conventional naming guidelines don't have an underscore in the `Test` method names (i.e. `TestNew` instead of `Test_New`). The exception are examples to distinguish between different variants (e.g. `ExampleNew` and `ExampleNew_someVariant`)

Reply:
Right! I copied from a wrong template 🤦‍♂